src/ui/pages/config/ToolbarPageConfig.js

```diff
@@ -16,6 +16,10 @@ class ToolbarPageConfig extends Component {
     this.props.onWillMount();
   }
 
+  componentWillUnmount() {
```

Comment:
Please let's avoid adding `componentWillMount` since it's deprecated. Ensure your logic works with `componentDidMount `.

Reply:
Sorry I think you overlooked that it's `componentWillUnmount()`, not `componentWillMount()`.

Reply:
Correct, sorry!